src/events/crawler/scrapers/USA/VA/index.js

```diff
@@ -126,25 +125,25 @@ const scraper = {
     'Rockbridge County',
     'Rockingham County',
     'Russell County',
-    'Salem city',
+    'Salem County',
     'Scott County',
     'Shenandoah County',
     'Smyth County',
     'Southampton County',
     'Spotsylvania County',
     'Stafford County',
-    'Staunton city',
-    'Suffolk city',
+    'Staunton County',
+    'Suffolk County',
     'Surry County',
     'Sussex County',
     'Tazewell County',
-    'Virginia Beach city',
+    'Virginia Beach County',
     'Warren County',
     'Washington County',
-    'Waynesboro city',
+    'Waynesboro County',
     'Westmoreland County',
-    'Williamsburg city',
-    'Winchester city',
```

Comment:
Ok, so these are actually counties, not cities? Or are they city-counties (same name for both city and county)?

Reply:
Some of them are city-counties while others are different counties and cities with the same name.